### DISCUSSION TITLE
搜索历史记录的api错误

### DISCUSSION THREAD

**Discussion by DhG314** [ywmoyue/biliuwp-lite#894] [bug]
### Bug 描述

api应该是"api.bilibili.com/x/web-interface/history/search"，而不是"api.bilibili.com/x/web-goblin/history/search"

### 操作步骤

```plain text
打开应用，进入历史记录页面，搜索任意内容
```

### 期望结果

返回正常的搜索结果

### 截图

![2024-10-28_102856](https://github.com/user-attachments/assets/ca15bac3-64d6-48aa-a3b0-3bdd79d80f63)
![2024-10-28_102739](https://github.com/user-attachments/assets/5ca3eaf7-1045-4dcb-bc08-9704f4de0612)

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
感谢反馈，之前是可以的，可能接口变了
